autobuild.xml

```diff
@@ -85,7 +84,6 @@
           <key>manifest</key>
           <array>
             <string>lib/release/*.so*</string>
-            <string>lib/debug/*.so*</string>
```

Comment:
What's the reasoning for removing the debug binaries?

Reply:
Artifacts left over from when this build script was used for alchemy.